.ci.gemfile

```diff
@@ -74,7 +74,13 @@ platform :ruby do
   gem 'rinku' # dependency for wikicloth for handling links
 
   gem 'RedCloth'
-  gem 'commonmarker', '< 1'
+
+  if ENV['LATEST_COMMONMARKER'] == 'yes'
```

Comment:
Is this necessary?  `commonmarker` uses `required_ruby_version`, so Ruby versions < 3.1 should automatically use an older version.  If you really need to force a version, something like:

```
if RUBY_VERSION >= '3.1'
  gem 'commonmarker'
else
  gem 'commonmarker', '< 1'
end
```

should work.

Reply:
I thought anyone might want to use less than version 1 of commonmarker with Ruby 3.1 or higher. Is this unnecessary?

Reply:
Certainly users can use commonmarker < 1 on Ruby >= 3.1.  However, we don't need to test that in CI.  I want CI to be reasonably small and fast, with no more than one run per Ruby version.